src/collects/seashell/backend/asan-error-parse.rkt

```diff
@@ -0,0 +1,344 @@
+#lang typed/racket
+
+(require/typed json [jsexpr->string (JSExpr -> String)]
```

Comment:
(require typed/json) gives you all these types, so you don't need to mess around with this.